spec/models/like_spec.rb

```diff
@@ -1,5 +1,5 @@
 require 'rails_helper'
 
 RSpec.describe Like, type: :model do
-  pending "add some examples to (or delete) #{__FILE__}"
+  # pending "add some examples to (or delete) #{__FILE__}"
```

Comment:
Currently, you have no unit specs for the `Like` model, remember that you should have unit specs for all of your models' methods and validations in order to accomplish all the requirements. Please add the necessary code in order to do this.